_opcodes/op352-pstee.html

```diff
@@ -0,0 +1,20 @@
+---
+n: 352
+opname: "Change Background"
+param1: "Irrelevant"
+param2: "Irrelevant"
+bg1: 0
+bg2: 0
+bgee: 0
+iwd1: 0
+iwd2: 0
+pst: 0
+pstee: 1
+---
+Sets the BMP file specified by the <code>resource</code> field as background image for the duration of the effect.
+
+{% capture note %}
+This opcode appears to work only if cutscene mode is active, either via <a href="../scripting/actions/psteeactions.htm#121">StartCutSceneMode()</a> or by <a href="#op356">opcode 356</a>.
+{% endcapture %}
+
+{% include note.html %}
```

Comment:
heh, can confirm — seems like Avenger ported this from gemrb. He added it there to more cleanly deal with the desert hell projectile.